src/main/java/io/camunda/zeebe/bpmnassert/testengine/PartitionCommandSenderImpl.java

```diff
@@ -0,0 +1,35 @@
+package io.camunda.zeebe.bpmnassert.testengine;
+
+import io.camunda.zeebe.engine.processing.message.command.PartitionCommandSender;
+import io.camunda.zeebe.engine.processing.message.command.SubscriptionCommandMessageHandler;
+import io.camunda.zeebe.util.buffer.BufferWriter;
+import java.util.Map;
+import org.agrona.concurrent.UnsafeBuffer;
+
+public class PartitionCommandSenderImpl implements PartitionCommandSender {
+
+  private final Map<Integer, SubscriptionCommandMessageHandler> subscriptionHandlers;
+
+  public PartitionCommandSenderImpl(
```

Comment:
:wrench: Refactoring proposal

We know in advance that we have only one partition. So we could pass in the handler directly and simplify the call sites leading up to here.